app/views/docs/query.phtml

```diff
@@ -0,0 +1,276 @@
+<p>
+Many Appwrite endpoints allows you to filter returned results through queries. Appwrite's SDKs include a <span class="tag">Query</span> class that helps construct query strings. This page documents all supported operations for Client SDKs. Server SDKs will behave similarly. 
+</p>
+
+<h2><a href="/docs/query#equal" id="equal">Equal</a></h2>
+<p>
+Returns results where the specified <b>attribute</b> is exactly equal to <b>value</b>.
+</p>
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Web</h3>
+        <div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Query } from "appwrite";
+const queryString = Query.equal(attribute, value);
+console.log(queryString);</code></pre>
```

Comment:
I see that we're printing the query string in each case, it seems unnecessary unless we show what the output looks like ... 

I think we can use two approaches here 

```js
import { Query } from "appwrite";

let attribute = "name";
let value = "Avengers";

const queryString = Query.equal(attribute, value);
console.log(queryString); // name=Avengers 
```

Reply:
In all these examples , we should mention that `value` can either be a single value like `Avengers` or a list of values like `['Avengers', 'Titanic']`

Reply:
Reference : https://github.com/appwrite/sdk-for-web/blob/master/src/sdk.ts#L3748-L3767